adapters/dogstatsd/dogstatsd.go

```diff
@@ -32,8 +33,11 @@ type dogstatsdAdapter struct {
 }
 
 func (a *dogstatsdAdapter) Stream(logstream chan *router.Message) {
+	var logspout_dogstatsd_enabled = os.Getenv("LOGSPOUT_DOGSTATSD_ENABLED")
```

Comment:
They tend to make use of a `getopt` function, though they seem to define it in multiple places.  Maybe worth a refactor?

https://github.com/remind101/logspout/blob/master/logspout.go#L15